plugins/linguist-backend/config.d.ts

```diff
@@ -14,11 +14,34 @@
  * limitations under the License.
  */
 
+import { TaskScheduleDefinition } from '@backstage/backend-tasks';
 import { HumanDuration } from '@backstage/types';
 
 export interface Config {
   /** Configuration options for the linguist plugin */
   linguist?: {
+    schedule?: TaskScheduleDefinition;
+    /**
+     * @default 20
+     */
+    batchSize?: number;
+    /**
+     * @default false
+     */
+    useSourceLocation?: boolean;
+    /**
+     * Refresh generated language breakdown
+     */
+    age?: HumanDuration;
+    /**
+     * @default ['API', 'Component', 'Template']
+     */
+    kind?: string[];
+    /**
+     * [linguist-js](https://www.npmjs.com/package/linguist-js) options
+     */
+    linguistJsOptions: object;
```

Comment:
```suggestion
    linguistJsOptions?: object;
```

you can also make this a `JsonObject` from `@backstage/types` to make it even more explicit if you want